osbs/api.py

```diff
@@ -622,8 +623,9 @@ def create_orchestrator_build(self, *args, **kwargs):
         kwargs.setdefault('outer_template', ORCHESTRATOR_OUTER_TEMPLATE)
         kwargs.setdefault('customize_conf', ORCHESTRATOR_CUSTOMIZE_CONF)
 
+        kwargs.setdefault('git_branch', None)
```

Comment:
So we're defaulting to `None`? I thought it needed to be defaulted to something like `unknown`, or did I misunderstand the problem?